cmd/temporalite/main.go

```diff
@@ -143,7 +153,17 @@ func buildCLI() *cli.App {
 				}
 
 				if c.IsSet(ipFlag) {
-					opts = append(opts, temporalite.WithFrontendIP(c.String(ipFlag)))
+					ip := c.String(ipFlag)
+					opts = append(opts, temporalite.WithFrontendIP(ip))
+					uiOpts.Host = ip
+				}
+				// TODO(jlegrone): allow configuration of web UI port
+				// Right now it doesn't seem possible to configure the web UI port,
+				// so in order to avoid port conflicts when running multiple Temporalite
+				// instances we'll only enable the web UI when the frontend port is not
+				// configured.
+				if !c.IsSet(portFlag) {
+					opts = append(opts, temporalite.WithUI(uiserver.NewServer(uiserveroptions.WithConfig(&uiOpts))))
```

Comment:
This appears to be working in https://github.com/temporalio/ui-server/pull/75 because `VITE_API=""` is set in the build script.